tests/data/placebo/test_elb_attribute_filter/elasticloadbalancing.DescribeLoadBalancerAttributes_1.json

```diff
@@ -0,0 +1,34 @@
+{
+    "status_code": 200, 
+    "data": {
+        "ResponseMetadata": {
+            "RetryAttempts": 0, 
+            "HTTPStatusCode": 200, 
+            "RequestId": "84ef0f17-5ab1-11e7-a809-a30bb7d62c26", 
+            "HTTPHeaders": {
+                "x-amzn-requestid": "84ef0f17-5ab1-11e7-a809-a30bb7d62c26", 
+                "date": "Mon, 26 Jun 2017 20:53:33 GMT", 
```

Comment:
Did you freshly record these? If not, please see https://cloudcustodian.io/docs/developer/tests.html#writing-placebo-tests-for-aws-resources for a guide on how to easily record in the future. When recording `        "ResponseMetadata": {}` and accountIds are automatically corrected so it can save some work.

Reply:
I did not! I just copy/pasted them. Thanks for the link, I'll address tonight.